app/grants/templates/grants/cancel.html

```diff
@@ -35,87 +36,100 @@
         </div>
       </div>
     </div>
-    <img src="{% static "v2/images/bitmap.png" %}" width="100%" />
+
+    <div class="alpha-warning font-caption">
+      <div>
+        {% trans "WARNING: Gitcoin Grants is in" %}
+        <span class="font-weight-bold">ALPHA</span>
+        {% trans "release." %}
+      </div>
+      <div>
+        {% trans "It has had inital audits done on it's smart contract and we are in the process of testing it holistically. Please, use caution and do not send high amounts of value." %}
+      </div>
+    </div>
 
     <div class="row">
       <div id="robot_error" class="hidden my-5">
         <img src="{% static "v2/images/prime.png" %}">
       </div>
     </div>
 
-    <div id="grants_form">
-      <h2 class="my-5 text-center">
-        <div class="mb-2 font-title-lg">
-          {% trans "Cancel contribution for" %}
-          <a href="{{ grant.reference_url }}" target="_blank">
-            {{ grant.title }}
-          </a>
-          {% trans "Grant" %}
+    <div class="row">
+
+      <div class="col-12 text-center mt-md-3">
+        <h2 class="font-title-lg my-4">{% trans "Cancel Contribution" %}</h2>
+      </div>
+
+      <div class="col-12 grant-banner py-5 row mx-0">
+        <div class="col-12 offset-md-2 col-md-3 py-4 banner-img">
+          <img src="{% if grant.logo and grant.logo.url %}{{ grant.logo.url }}{% else %}{% with grant_logo='v2/images/grants/logos/' id=grant.id|modulo:3|add:1 %} {% static grant_logo|addstr:id|add:'.png' %} {% endwith %} {% endif %}">
```

Comment:
So since we weren't saving an image if the grant owner doesn't add it! We've map it to one of the 4 static assets (default images) we have.
to ensure the same default always loads -> I just go `grant_id % 3  + 1` to map it to the file 

I know it's meh 😓 could use some help with that. 

but ideally that code shows this 

<img width="1412" alt="screen shot 2018-11-26 at 11 00 23 pm" src="https://user-images.githubusercontent.com/5358146/49031243-6f24b100-f1cf-11e8-9b35-19e0272435a0.png">

Reply:
i really like the design.  but still think the logic for what image is shown could be encapsulated into a method off of the grants object.